addons/hr/models/hr_department.py

```diff
@@ -42,6 +45,13 @@ def _compute_complete_name(self):
             else:
                 department.complete_name = department.name
 
+    @api.depends('parent_path')
+    def _compute_master_department_id(self):
+        # Don't use the cache as the value is updated in SQL
```

Comment:
This comment implies the cache is not correctly invalidated on update to `parent_path`, wouldn't that also mean `depends` will not work properly?